frontend/src/types/types.ts

```diff
@@ -83,3 +84,9 @@ export interface Bar {
   color?: string;
   tooltipText?: string;
 }
+
+export interface State {
```

Comment:
Suggestion: I think `State` is not descriptive enough. We could use sth like this:
```suggestion
export interface SegmentState {
```